api/v1alpha1/annotations.go

```diff
@@ -4,9 +4,15 @@ const (
 	// AnnotationKeyPrefix is the prefix used for annotation key values
 	AnnotationKeyPrefix = LabelKeyPrefix
 
-	// SSOUserIDAnnotationKey is used to store the user's user_id claim value issued by the SSO provider
-	SSOUserIDAnnotationKey = AnnotationKeyPrefix + "sso-user-id"
+	// UserIDUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
+	// contains the user's User ID as set in the user's JWT token.
+	UserIDUserAnnotationKey = AnnotationKeyPrefix + "sso-user-id"
 
-	// SSOAccountIDAnnotationKey is used to store the user's account_id claim value issued by the SSO provider
-	SSOAccountIDAnnotationKey = AnnotationKeyPrefix + "sso-account-id"
+	// AccountIDUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
+	// contains the user's Account ID as set in the user's JWT token.
+	AccountIDUserAnnotationKey = AnnotationKeyPrefix + "sso-account_id"
+
+	// EmailUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
+	// contains the user's Email as set in the user's JWT token.
+	EmailUserAnnotationKey = AnnotationKeyPrefix + "email"
```

Comment:
The original annotation used `LabelKeyPrefix + "user-email"` key string. Are you sure that you want to change it?
```suggestion
	// EmailUserAnnotationKey is used to set an annotation value in the User resource on the member cluster, that
	// contains the user's Email as set in the user's JWT token.
	EmailUserAnnotationKey = AnnotationKeyPrefix + "user-email"
```

Reply:
This was intentional, it had actually been planned quite a long time ago as it's more technically correct.